tests/testFa2Interface.ml

```diff
@@ -264,20 +267,20 @@ let suite =
     (* TODO: ledger_*_liquidity functions do not perform any permission check.
      * This we can only check at a higher level, probably with e2e tests. *)
```

Comment:
Ah yes, this is still a pretty valid TODO here. The unit tests in `tests/testFa2Interface.ml` only test the functionality of functions declared in `fa2Interface.ml`. However, I don't know what kind of tests could we add for permissions, since these functions are essentially called by checker in several places, not in a single entrypoint. I think it might be best to just add unit tests in `tests/testChecker.ml` for `Checker.strict_entrypoint_transfer`, which now we have none of.

FWIW I believe that every unit test in `tests/testChecker.ml` could probably be turned into an e2e test, but we might not want to do that, given that it takes some minutes to run e2e tests :sweat_smile: